modules/ppcp-onboarding/src/OnboardingModule.php

```diff
@@ -48,13 +48,7 @@ public function run( ContainerInterface $c ): bool {
 		add_action(
 			'admin_enqueue_scripts',
 			function() use ( $c ) {
-				if (
-					apply_filters(
-					// phpcs:ignore WordPress.NamingConventions.ValidHookName.UseUnderscores
-						'woocommerce.feature-flags.woocommerce_paypal_payments.settings_enabled',
-						getenv( 'PCP_SETTINGS_ENABLED' ) === '1'
-					) && ! SettingsModule::should_use_the_old_ui()
-				) {
+				if ( ! SettingsModule::should_use_the_old_ui() ) {
```

Comment:
I think it would be better to rely on the filter than on the static method because the new settings module is not added to the container for existing merchants.

So in the conditional I suggest to remove the call to the static method and use just the filter:
```php
if(
   apply_filters(
      'woocommerce.feature-flags.woocommerce_paypal_payments.settings_enabled',
      '1' === get_option( 'woocommerce-ppcp-is-new-merchant' ) 
      || getenv( 'PCP_SETTINGS_ENABLED' ) === '1'
	)
)
```